src/components/GroupGraph/GroupGraph.module.css

```diff
@@ -0,0 +1,48 @@
+
+.hint {
+    font-size: var(--fs-small-text)
+}
+
+.sorters {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 5rem;
+    margin: 3.5rem;
+}
+
+.sort {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+@media (max-width: 45em) {
+    .sorters {
+        gap: 2rem;
+    }
+}
+
+
+/* GRAPH */
+
+.graphs {
+    display: grid;
+    grid-template-columns: 1fr 3fr;
+    grid-gap: 4rem 0rem;
+    justify-content: center;
+    align-items: center;
+}
+
+.graph-title {
+    text-align: left;
+    font-size: var(--fs-mid-small-text);
+    display: flex;
+    justify-content: center;
```

Comment:
<img width="1377" alt="Screenshot 2023-05-29 at 12 38 50" src="https://github.com/Code4UHub/client/assets/85084188/02f781d7-51e7-48e4-af47-9e221f035373">
Is there a way to align the graph title and description to its content?